src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -312,6 +313,20 @@ public void Initialize(IConsole console)
                         }
 
                         UpdateWorkingDirectory();
+
+                        if (!PowerShellLoaded)
+                        {
+                            var telemetryEvent = new TelemetryEvent("PowerShellLoaded", new Dictionary<string, object>
+                            {
+                                { "Trigger", console is IWpfConsole ? "PMC" : "PMUI" }
```

Comment:
Use a constant or readonly string:

```suggestion
                                { NuGetPowerShellUsageCollector.Trigger, console is IWpfConsole ? NuGetPowerShellUsageCollector.PMC : NuGetPowerShellUsageCollector.PMUI }
```

Reply:
Addressed.